lib/pbench/server/auth/__init__.py

```diff
@@ -225,6 +225,7 @@ def wait_for_oidc_server(
         )
         adapter = HTTPAdapter(max_retries=retry)
         session.mount("http://", adapter)
+        session.mount("https://", adapter)
```

Comment:
Did you mean to change/replace the old `http://` mount?

Reply:
Originally, I replaced it; then it occurred to me that we're likely to have an interval here where we access the OIDC server via `http`, in which case we probably want _both_ mounts.  😛